src/Utils/api.js

```diff
@@ -23,3 +23,14 @@ export const getArticleById = (article_id) => {
       return err;
     });
 };
+
+export const getCommentsByArticleId = (article_id) => {
+  return api
+    .get(`/articles/${article_id}/comments`)
+    .then((res) => {
+      return res.data.comments;
+    })
+    .catch((err) => {
```

Comment:
This catch block is not necessary as you have one in your CommentsList component.